docbook/reference/en/en-US/modules/tracing.xml

```diff
@@ -0,0 +1,278 @@
+            of information.
+        </para>
+        <para>
+            Currently it doesn't have a standard or spec to define the tracing feature, so the tracing feature is
+            tightly coupled with the concrete JAX-RS implementation itself. In this chapter, let's check the design and
+            usage of the tracing feature.
+        </para>
+    </section>
+    <section>
+        <title>Tracing Info Mode</title>
+        <para>The RESTEasy tracing feature supports three logging mode:
+        </para>
+        <itemizedlist>
+            <listitem>OFF</listitem>
+            <listitem>ON_DEMAND</listitem>
+            <listitem>ALL</listitem>
+        </itemizedlist>
+        <para>
+            "ALL" will enable the tracing feature. "ON_DEMAND" mode will give the control to client side: A client can
+            send
+            a tracing request via HTTP header and get the tracing info back from response headers. "OFF" mode will
+            disable
+            the tracing feature, and this is the default mode.
+        </para>
+    </section>
+    <section>
+        <title>Tracing Info Level</title>
+        <para>The tracing info has three levels:
+        </para>
+        <itemizedlist>
+            <listitem>SUMMARY</listitem>
+            <listitem>TRACE</listitem>
+            <listitem>VERBOSE</listitem>
+        </itemizedlist>
+        <para>
+            The "SUMMARY" level will emit some brief tracing information. The "TRACE" level will produce more detailed
+            tracing information, and the "VERBOSE" level will generate extremely detailed tracing information.
+        </para>
+        <para>
+            The tracing feature relies on the JBoss Logging framework to produce the tracing info, so the JBoss Logging
+            configuration actually controls the final output of the tracing info. So it is JBoss Logging framework
+            configuration
+            that controls the logging threshold of the tracing info.
+        </para>
+    </section>
+    <section>
+        <title>Basic Usages</title>
+        <para>By default, the tracing feature is turned off. If you want to enable the tracing feature, you need to
+            add the following dependency in your project:
+            <programlisting><![CDATA[<dependency>
+    <groupId>org.jboss.resteasy</groupId>
+    <artifactId>resteasy-tracing-api</artifactId>
+</dependency>]]></programlisting>
+        </para>
+        <para>
+            Because the tracing feature is an optional feature, the above dependency is provided by the
+            <ulink url="https://github.com/resteasy/resteasy-extensions">resteasy-extensions</ulink>
+            project.
+        </para>
+        <para>
+            After including the dependency in your project, you can set the tracing mode and tracing level via the
+            context-param parameters in your web
```

Comment:
Does this use the MP Config additions that @ronsigal did?

Reply:
@asoldano I think so: https://github.com/resteasy/Resteasy/blob/master/resteasy-core/src/main/java/org/jboss/resteasy/core/ResteasyDeploymentImpl.java#L108

Reply:
@liweinan ok, great, so you should generally speaking of configuration parameters here, instead of context-params from web.xml. Please check elsewhere in the doc for similar references.